aws-test/tests/aws_elasticache_cluster/test-list-query.sql

```diff
@@ -0,0 +1,3 @@
+select akas, cache_cluster_id, cache_node_type, engine, title
+from aws.aws_elasticache_cluster
+where cache_cluster_id = '{{ output.resource_id.value }}';
```

Comment:
Use `arn` instead of using `cache_cluster_id` for list call, as `cache_cluster_id` is used for GET.